src/components/HotRepoCard.tsx

```diff
@@ -26,8 +29,19 @@ const HotRepoCard = ({ repoName }: HotRepoCardProps): JSX.Element => {
 
   if (isError) {
     return (
-      <div className="p-4 border rounded-2xl bg-white w-full space-y-1 relative">
-        {`${repoName} failed to load`}
+      <div className="p-4 border rounded-2xl bg-white w-full space-y-1 relative flex flex-col justify-between">
+        {`⚠️ ${repoName} repo could not be loaded`}
+
+        <div className="flex justify-center">
+          <a
+            className="bg-cheesyYellow text-grey rounded-xl font-bold hover:text-saucyRed transition-all duration-300 mr-3 p-2 flex w-5/6 h-fit justify-center"
+            href={`${bugReportLink} repo not found&body=Please take a look why this ` + `${repoName}` + `not founded`}
```

Comment:
```suggestion
            href={`${bugReportLink}?title=${title}&body=${body(repoName)}`}
```

This does not seem ideal. You are combining string interpolation and string literals. Also the title is part of the report link, but the body is not.